apps/www/scripts/build-registry.mts

```diff
@@ -285,14 +290,23 @@ async function buildStyles(registry: Registry) {
       }
 
       const files = item.files?.map((file) => {
-        const content = readFileSync(
-          path.join(process.cwd(), "registry", style.name, file),
-          "utf8"
-        )
-
-        return {
-          name: basename(file),
-          content,
+        switch (file.split(".")[1]) {
+          case "stories":
+            return {
+              name: basename(file),
+              content: readFileSync(
+                path.join(process.cwd(), "registry", file),
+                "utf8"
+              ),
+            }
+          default:
+            return {
+              name: basename(file),
+              content: readFileSync(
+                path.join(process.cwd(), "registry", style.name, file),
+                "utf8"
+              ),
+            }
```

Comment:
Similar to the switch-statement in the `buildRegistry` function, there might be some room for refactoring here to make more DRY, but am not seeing a huge need with only two cases atm.